src/app/shared/constants/options/creature-template-spell-index.ts

```diff
@@ -0,0 +1,12 @@
+import { Option } from '../../types/general';
+
+export const CREATURE_TEMPLATE_SPELL_INDEX: Option[] = [
+  { value: 0,   name: '' },
+  { value: 1,   name: '' },
+  { value: 2,   name: '' },
+  { value: 3,   name: '' },
+  { value: 4,   name: '' },
+  { value: 5,   name: '' },
+  { value: 6,   name: '' },
+  { value: 7,   name: '' },
```

Comment:
o_O what is this? why names are empty?

Reply:
Because they are only indexes, with no names.
But as it can only be 0-7 I prefer providing that

Reply:
then why making a selector at all? should be more of a tooltip saying "Allowed values: 0-7"

Reply:
done